src/test/java/com/johnstarich/moviematcher/controllers/MovieServiceTest.java

```diff
@@ -77,25 +76,38 @@ public void testGetMovieSearchResultsForGarbage() throws Exception {
 		Session s = login("Welcome1");
 		get(movieService.PREFIX + "/search/dllsakdjflaiwef;alsdkhlueiahe", authHeaders(s), response -> {
 			Map<String, Object> returnedMap = response.json(HashMap.class);
-			List<LinkedTreeMap> movieResults = (List) returnedMap.get("movies");
-			HashSet<String> movieTitles = new HashSet<>();
-			movieResults.forEach(m -> movieTitles.add(m.get("title").toString()));
-
-			assertTrue(movieTitles.isEmpty());
+			List<LinkedTreeMap<String, String>> movieResults = (List) returnedMap.get("movies");
+			assertTrue(movieResults.isEmpty());
 		});
 	}
-	public void testGetMovieByMovieId() throws Exception {
+	public void testGetMovieWithMovieId() throws Exception {
+		ObjectId movieId = new ObjectId(); 
+		Movie theDarkKnight = addMovie(movieId, "The Dark Knight"); 
+		register("Welcome1"); 
+		Session s = login("Welcome1"); 
+		get(movieService.PREFIX + "/" + movieId, authHeaders(s), response -> {
+			Movie movieResult = response.json(Movie.class); 
+			assertEquals(theDarkKnight.title, movieResult.title); 
+			assertEquals(theDarkKnight.id, movieResult.id); 
+		}); 
+	}
+    public void testGetMovieWithInvalidMovieId() throws Exception {
```

Comment:
I recommend highlighting the whole file, in IntelliJ go to "help" and search for "tab" and find the convert to tabs option